src/lib/openapi/util/openapi-tags.ts

```diff
@@ -18,6 +18,11 @@ const OPENAPI_TAGS = [
         description:
             'Create, update, and delete [Personal access tokens](https://docs.getunleash.io/reference/api-tokens-and-client-keys#personal-access-tokens).',
     },
+    {
+        name: 'Service Accounts',
+        description:
+            'Endpoints for managing [Service Accounts](https://docs.getunleash.io/reference/service-accounts) enabling apps to access Unleash API programatically.',
```

Comment:
Small typo in `programatically` - `programmatically`.

Small nitpick but the sentence doesn't read very well to me, maybe something like this instead?

```suggestion
            'Endpoints for managing [Service Accounts](https://docs.getunleash.io/reference/service-accounts), which enable programmatic access to the Unleash API.',
```